frontend-react/src/components/USLink.test.tsx

```diff
@@ -55,3 +61,48 @@ describe("USNavLink", () => {
     // value. It's assumed that react-router-dom is testing that as a part of
     // their package.
 });
+
+describe("USLinkButton", () => {
+    test("boolean button styles applied", () => {
+        const view = renderWithRouter(
+            <USLinkButton secondary base outline inverse unstyled>
+                Test
+            </USLinkButton>
+        );
+        expect(view.container.children[0]).toHaveClass(
```

Comment:
Small point, but can this be found with a `getByText` or even `getByRole`?   If there's something visual to be queried for, I think we should do that instead of relying on the return renderer structure (even though that's unfortunately necessary sometimes 😬)